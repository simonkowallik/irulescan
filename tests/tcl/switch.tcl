
switch "abc" {
    abc {}
    default {expr 2}

}

#switch abc {
#    a -
#    b {expr 1}
#    $foo {expr 2}
#    default {expr 3}
#}


# switch abc a {expr 1}
# switch abc a - b {expr 1} $foo {expr 2} default {expr 3}
# switch -- abc a - b {expr 1} $foo {expr 2} default {expr 3}
# switch -- abc { a - b {expr 1} $foo {expr 2} default {expr 3} }
