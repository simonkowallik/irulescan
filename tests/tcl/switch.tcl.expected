DANGEROUS: missing options terminator at `--` in `switch "abc" {    abc {}    default {expr 2}}`
WARNING: Cannot scan switch, not implemented (TODO, known-issue). at `` in `switch "abc" {    abc {}    default {expr 2}}`
WARNING: badly formed command at `switch` in `switch "abc" {    abc {}    default {expr 2}}`

